modules/installation-gcp-marketplace.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-marketplace-_{context}"]
+= Using a GCP Marketplace image
+If you are deploying an {product-title} cluster using a GCP Marketplace image, create the manifests and edit the compute machine set definitions to specify the GCP Marketplace image.
```

Comment:
I can't make a better suggestion, but the "If you are deploying..." sounds off to me. I'm assuming that they technically haven't done it yet, and this is a step they need to do prior to deploying, if they want to use a GCP Marketplace image?

I don't know if either of these is any better (or still accurate), but there are my thoughts/suggestions:
* If you want to deploy a {product-title} cluster using a GCP Marketplace image, you must create...
* To deploy a {product-title} cluster using a GCP Marketplace image, you must first create...

Reply:
> * If you want to deploy a {product-title} cluster using a GCP Marketplace image, you must create...

This works and remains technically accurate. Thank you for pointing this out.